plugins/modules/azure_rm_monitoractiongroups.py

```diff
@@ -0,0 +1,754 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2025 Klaas Demter (@Klaas-)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: azure_rm_monitoractiongroups
+version_added: "3.7.0"
+short_description: Create, update and delete Action Groups
+description:
+    - Create, update and delete Action Groups
```

Comment:
There should be a period at the end of each sentence.
```suggestion
    - Create, update and delete Action Groups.
```